internal/daemonset/daemonset.go

```diff
@@ -18,12 +18,13 @@ import (
 )
 
 const (
-	kubeletDevicePluginsVolumeName = "kubelet-device-plugins"
-	kubeletDevicePluginsPath       = "/var/lib/kubelet/device-plugins"
-	nodeLibModulesPath             = "/lib/modules"
-	nodeLibModulesVolumeName       = "node-lib-modules"
-	nodeUsrLibModulesPath          = "/usr/lib/modules"
-	nodeUsrLibModulesVolumeName    = "node-usr-lib-modules"
+	kubeletDevicePluginsVolumeName   = "kubelet-device-plugins"
+	kubeletDevicePluginsPath         = "/var/lib/kubelet/device-plugins"
+	nodeLibModulesPath               = "/lib/modules"
+	nodeLibModulesVolumeName         = "node-lib-modules"
+	nodeUsrLibModulesPath            = "/usr/lib/modules"
+	nodeUsrLibModulesVolumeName      = "node-usr-lib-modules"
+	nodeDriverContainerLabelTemplate = "oot.node.kubernetes.io/%s.ready"
```

Comment:
I don't think we need to have this as a constant, it's only used in `GetDriverContainerNodeLabel`

Reply:
fixed